gatsby-config.ts

```diff
@@ -1,3 +1,5 @@
+/*eslint camelcase: ["error", {properties: "never"}]*/
```

Comment:
We can take the camelCase rule out if it's causing issues for the css. I know that for bulma, it wll be easier to hypenate

Reply:
(Not to tackle in this PR - just a general callout)